src/libsync/theme.h

```diff
@@ -40,6 +40,8 @@ class OWNCLOUDSYNC_EXPORT Theme : public QObject
     Q_PROPERTY(bool branded READ isBranded CONSTANT)
     Q_PROPERTY(QString appNameGUI READ appNameGUI CONSTANT)
     Q_PROPERTY(QString appName READ appName CONSTANT)
+    Q_PROPERTY(QString currentAccountStateIndicatorOnlineImageSource READ currentAccountStateIndicatorOnlineImageSource)
```

Comment:
I'd follow the naming of the other related functions and properties in here like `folderOfflineIcon` and name this `folderOfflineSource` or similar. Carrying the "currentAccountState" naming in there makes it too specific to its consumption point while it's internal API.

Also since it doesn't change at runtime and doesn't have a NOTIFY part, please add a CONSTANT marker to that property. This allows some optimizations in the QML engine.

Same comment applies to the property just below.